mgr/src/cmds/volumes.cr

```diff
@@ -3,7 +3,8 @@ require "./volume_create_parser"
 require "./gluster_volume_parser"
 
 struct VolumeArgs
-  property status = false, detail = false, name = "", volume_id = "", no_start = false, auto_create_pool = false, auto_add_nodes = false,
+  property status = false, detail = false, name = "", volume_id = "", no_start = false,
+    auto_create_pool = false, auto_add_nodes = false, json = false,
```

Comment:
Add `json` to Args in helpers.cr(Refer `script_mode`)

Reply:
Done.